### PR TITLE
feat(webdriver-manager): update download logic with streaming

### PR DESCRIPTION
**Pull request by juliemr**
This seems to fix up some unreliable download issues that we were
previously having.

### CODE REVIEW

File: bin/webdriver-manager [hunk 1]
```diff
@@ -133,6 +133,7 @@ var httpGetFile = function(fileUrl, fileName, outputDir, callback) {
   var filePath = path.join(outputDir, fileName);
   var file = fs.createWriteStream(filePath);
   var ignoreSSL = argv.ignore_ssl;
+  var length = 0;
```
Comment (hankduan) [inline bin/webdriver-manager]: nit: maybe name this to contentLength

**Comment by hankduan** [comment]
LGTM

File: bin/webdriver-manager (hunk 1 above)
Comment (juliemr) [inline bin/webdriver-manager]: done.

**Comment by juliemr** [comment]
Thanks. Merged with aded26bc9ee6172d6f64361207f6a8b04da09b0d

**Pull request closed without merging**